variables.tf

```diff
@@ -36,6 +36,12 @@ variable "log_sink_name" {
   type        = string
 }
 
+variable "description" {
+  description = "A description of this sink. The maximum length of the description is 8000 characters."
+  type        = string
+  default     = ""
```

Comment:
@tanguynicolas is it possible to set default value null?

Reply:
I referred to other Google modules to choose the default value (e.g. [this module](https://github.com/terraform-google-modules/terraform-google-network/blob/master/variables.tf#L109)).

What rule do you use to choose between `null` and `""`?

Reply:
@tanguynicolas thx. Those were written before TF supported null values.